src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -75,7 +73,7 @@ const PageLayoutWrapper = ({
       }, 0);
   }
 
-  const serviceFonts = fontFacesLazy(service, isPWA);
+  const serviceFonts = fontFaces();
 
   const fontJs =
```

Comment:
Just an aside, we should really move this code into it's own file with accompanying unit tests at some point, having this all inline in a template literal

Reply:
I've added it to that doc!